build.sbt

```diff
@@ -88,9 +87,7 @@ val `scalajs-bundler` =
         commitNextVersion,
         pushChanges,
         releaseStepTask(GhPagesKeys.pushSite in manual)
-      ),
-      scriptedLaunchOpts += "-Dplugin.version=" + version.value,
-      scriptedBufferLog := false
```

Comment:
Not sure why the top-level project contains settings for the scripted plugins, so I've removed them.

Reply:
If I remember correctly that was needed to run the `scripted` command from the root project. I noticed that you use the `runScripted` now instead. What’s the difference between them?

Reply:
Completely missed this comment. I will update CONTRIBUTNG.md to mention the new `runScripted` task. A future improvement could be to run the supported scripted tests as part of the normal test target

```
test in Test := Def.sequential(
   test in Test,
   runScriptedTask
).value,
```

Reply:
Updated CONTRIBUTING.md to mention `runScripted`.